src/utils/daos_hdlr.c

```diff
@@ -2834,18 +2777,28 @@ fs_copy_hdlr(struct cmd_args_s *ap)
 	struct fs_copy_args	fa = {0};
 	int			src_str_len = 0;
 	int			dst_str_len = 0;
+	char			*name = NULL;
+	char			*dname = NULL;
+	char			dst_dir[MAX_FILENAME];
+	int			path_length = 0;
+	mode_t			tmp_mode_dir;
```

Comment:
Why not set this here, it looks to always be the same value?  Alternatively just use a define.